.github/workflows/build-test.yml

```
@@ -44,7 +44,7 @@ jobs:
     name: Build Development Image
     needs:
       - prepare-environment
-    uses: open-space-collective/open-space-toolkit/.github/workflows/build-image.yml@main
+    uses: open-space-collective/open-space-toolkit/.github/workflows/build-image.yml@build/update-github-actions-extensions
```

```suggestion
    uses: open-space-collective/open-space-toolkit/.github/workflows/build-image.yml@main
```
Apply once base MR is merged